lib/tuples.gi

```diff
@@ -506,3 +506,19 @@ InstallOtherMethod( \*,
     fi;
     return DirectProductElement( List( dpelm, entry -> nonlist * entry ) );
     end );
+
+
+#############################################################################
+##
+##
```

Comment:
Do you expect a need for additional methods? Otherwise, this could be just a global function...

Reply:
I've converted it into a global function.